src/backend/utils.ts

```diff
@@ -1070,38 +1061,63 @@ export async function moveOnUnix(
   if (rsyncExists) {
     const origin = install_path + '/'
     logInfo(
-      `moving command: rsync -az --progress ${origin} ${destination} `,
+      `moving command: rsync --archive --compress --no-human-readable --remove-source-files --info=name,progress ${origin} ${destination} `,
       LogPrefix.Backend
     )
     const { code, stderr } = await spawnAsync(
       'rsync',
-      ['-az', '--progress', origin, destination],
+      [
+        '--archive',
+        '--compress',
+        '--no-human-readable',
+        '--remove-source-files',
+        '--info=name,progress',
+        origin,
+        destination
+      ],
       { stdio: 'pipe' },
       (data) => {
-        const split =
-          data
-            .split('\n')
-            .find((d) => d.includes('/') && !d.includes('%'))
-            ?.split('/') || []
-        const file = split.at(-1) || ''
-
-        if (file) {
-          currentFile = file
+        let percent = 0
+        let eta = ''
+        let bytes = '0'
+        // Rsync outputs either the file currently being transferred or a
+        // progress report. To know which one of those `data` is, we check if
+        // it includes a %, :, and starts with a space
+        // If all of these aren't the case, it's *most likely* a filename
+        // FIXME: This is pretty hacky, but I don't see an obvious way to
+        //        "divide" the two output types other than that
+        const isFilenameOutput =
+          !data.includes('%') && !data.includes(':') && !data.startsWith(' ')
+
+        if (isFilenameOutput) {
+          // If we have a filename output, we've started copying a new
+          // file (we thus start at 0%, with 0 bytes moved, and with no ETA)
+
+          // Data will be a multiline string, potentially with directory names,
+          // finally the file name, and ending with an empty line
+          currentFile = data.trim().split('\n').at(-1)!
+        } else {
+          // If we got the progress update, try to read out the bytes, ETA and
+          // percent
+          const bytesMatch = data.match(/^\s*(\d+)/)?.[1]
+          const etaMatch = data.match(/(\d+:\d+:\d+)/)?.[1]
```

Comment:
```suggestion
          const etaMatch = data.match(/(\d+:\d\d:\d\d/)?.[1]
```

since we know it's always 2 digits for the second and minute